package.json

```diff
@@ -18,6 +18,7 @@
   "homepage": "https://bitbucket.org/valtechny/eslint-config-valtech#readme",
   "devDependencies": {
     "gulp": "^3.9.1",
-    "gulp-eslint": "^2.0.0"
+    "gulp-eslint": "^2.0.0",
+    "eslint-plugin-jsdoc": "^2.3.1"
```

Comment:
This shouldn't be a `devDependency`; otherwise, dependent packages won't install the required plugin when using.